src/CommunityToolkit.Maui.UnitTests/PopupServiceTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.ComponentModel;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.UnitTests.Mocks;
+using CommunityToolkit.Maui.Views;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests;
+
+public class PopupServiceTests : BaseHandlerTest
```

Comment:
We should unit test and prove that only the expected instances are created during Popup creation.